samples/ControlCatalog/Pages/DataGridPage.xaml

```diff
@@ -96,18 +96,18 @@
           <DataGrid Name="dataGridEdit" Margin="12" Grid.Row="0"
                     Items="{Binding DataGrid3Source}">
             <DataGrid.Columns>
-              <DataGridTextColumn Header="First Name" Binding="{Binding FirstName}" Width="2*" FontSize="{Binding #FontSizeSlider.Value, Mode=OneWay}" />
-              <DataGridTextColumn Header="Last" Binding="{Binding LastName}" Width="2*" FontSize="{Binding #FontSizeSlider.Value, Mode=OneWay}" />
-              <DataGridCheckBoxColumn Header="Is Banned" Binding="{Binding IsBanned}" Width="*" IsThreeState="{Binding #IsThreeStateCheckBox.IsChecked, Mode=OneWay}" />
+              <DataGridTextColumn Header="First Name" Binding="{Binding FirstName}" Width="2*" FontSize="{Binding #FontSizeSlider.Value, Mode=OneWay}" x:DataType="local:Person" />
```

Comment:
For some context, you don't need to specify x:DataType here, because compiler now inherits it from the Items property binding. Same as it did before for ListBox.
You might need to rebuild compiler projects.